src/Menu.php

```diff
@@ -0,0 +1,469 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bulma;
+
+use Closure;
+use Yiisoft\Arrays\ArrayHelper;
+use Yiisoft\Html\Html;
+
+use function array_merge;
+use function array_values;
+use function call_user_func;
+use function count;
+use function implode;
+
+final class Menu extends Widget
+{
+    private string $activeCssClass = 'is-active';
+    private bool $activateItems = true;
+    private bool $activateParents = false;
+    private string $brand = '';
```

Comment:
What's the brand in the menu?

Reply:
Title menu:

![image](https://user-images.githubusercontent.com/42547589/87177061-08231580-c2a9-11ea-9fb9-7f9b9b2ec35a.png)